experimental/dds/tree2/src/feature-libraries/editable-tree-2/proxies/proxies.ts

```diff
@@ -289,6 +286,7 @@ function addDescriptor(
 
 // These are methods implemented by LazySequence that we expose through the proxy.
 [
+	LazySequence.prototype.at,
```

Comment:
@DLehenbauer I'm still trying to wrap my mind around this file and what exactly it's doing, so unsure about this chang and wanted to double check. The fact that `LazySequence` already had an `at()` method through `LazyField` but this wasn't here makes me particularly unsure if it's correct.

Reply:
Outdated, new similar comment upcoming